pyvista/ext/plot_directive.py

```diff
@@ -344,7 +344,7 @@ def _split_code_at_show(text):
 
 # the context of the plot for all directives specified with the
 # :context: option
-plot_context = {}
+plot_context = {}  # type: ignore[var-annotated]
```

Comment:
Is this needed now?

Reply:
No, we can remove this, good catch
```suggestion
plot_context = {}
```